elements/analysis/aggcountervector.hh

```diff
@@ -133,7 +133,7 @@ AggregateCounterVector::find_node(uint32_t a, const Packet* p, bool &outdated)
         struct rte_mbuf* p_mbuf;
         p_mbuf = (struct rte_mbuf *) p->destructor_argument();
 
-        if (!(p_mbuf->ol_flags & PKT_RX_FDIR_ID))  {
+        if (!(p_mbuf->ol_flags & RTE_MBUF_F_RX_FDIR_ID))  {
```

Comment:
This will break the compilation with any older DPDK, we want to keep compatibility. Please enclose all changes in an ifdef as I showed you on teams, or globally define an alias.